### PR TITLE
Started adding credits to course data base

### LINKED ISSUE #231: Data - Augment API to hold credit information
**Job**
When I schedule for next semester, I want to know how many credits the schedule is going to be.

**Describe a possible solution**
In each of the example csv files like `rpi_data/summer-2020.csv` it has a column named `course_credit_hours`, include this so that `min_credits` and `max_credits` exist for new course records in the db (src/data/schema/01_course.sql).

This should be exposed in the API as well through the `/class` queries in open rpi_data/summer-2020.csv

**Context**
We have the data in the data csv's, but it is not propagated up to the. First augment the API, then we'll create separate issues on populating the Scheduler and Explore views.

**To Test**
Verify the api call for /class has the new fields for min and max credits

### PR DESCRIPTION
**Pull request by gizzon**
*PR*

**Issue**
closes #231 

**Database Changes/Migrations**

Created two database attributes on the course and course session table: min_credits and max_credits. These are responsible for holding the minimum and maximum amount of credits a class can be taken for. This PR is strictly backend. The credits changes cannot be seen on yacs yet, but the credits are now available in the tables and API.

### CODE REVIEW

**Comment by 123joshuawu** [comment]
@gizzon Not sure if you managed to resolve the docker blocker, but to remove the volume for postgres, you can run `docker volume rm yacsn_postgres_data`

**Comment by YuetingLiao** [comment]
Great! The front-end needs to display credit information on subject/course. I was just able to ask for the db attribute for course object(table).

File: src/api/db/courses.py [hunk 1]
```diff
@@ -160,6 +164,7 @@ def populate_from_csv(self, csv_text):
                                 "Title": row['course_name'],
                                 "RawPrecoreqText": row['raw_precoreqs'],
                                 "School": row['school']
+
```
Comment (jshom) [inline src/api/db/courses.py:167]: remove extraneous newline

**Review by jshom — APPROVED** [review]
remove newline and lgtm!

**Review by venkatrsrinivas — APPROVED** [review]
Hello. 
This all looks great to me and I don't see anything that needs any additional adjustments. 
It'll be interesting to see the Credits Displayed on the Frontend Side soon.

**Pull request merged**